draft-packages/guidance-block/KaizenDraft/GuidanceBlock/GuidanceBlock.scss

```diff
@@ -102,15 +101,48 @@ $ca-banner-collapse-height-delayed: margin-top $ca-duration-fast
 }
 
 .headingWrapper {
-  margin-bottom: $ca-grid / 4;
+  margin-bottom: $kz-var-spacing-md / 4;
 }
 
 .cancel {
-  @include button-reset;
-  composes: interactiveIconWrapper from "~@kaizen/component-library/components/Icon/Icon.module.scss";
+  appearance: none;
+  background: transparent;
+  border: none;
+  font: inherit;
+  margin: 0;
+  padding: 0;
+  transition: none;
+
+  cursor: pointer;
+  .icon {
+    opacity: 0.7;
+    transition: $kz-var-animation-duration-fast opacity;
+  }
+  &:disabled,
+  &.disabled {
+    .icon {
+      opacity: 0.3;
+    }
+  }
+  &:not(:disabled):not(.disabled) {
+    &:hover,
+    &:focus,
+    &.hover {
+      .icon {
+        opacity: 1;
+      }
+    }
+    &:active,
+    &.active {
+      .icon {
+        opacity: 1;
+      }
+    }
+  }
```

Comment:
Is there no button reset mixin anymore?

Reply:
There is but I'm trying to reduce the number of stylesheet dependencies because that seems to cause a lot of bugs with kaizen.